TwipsterApp/LoginWindow.xaml.cs

```diff
@@ -7,15 +11,41 @@ namespace TwipsterApp
     /// </summary>
     public partial class LoginWindow : Window
     {
+        public static User currentUser;
         public LoginWindow()
         {
             InitializeComponent();
         }
 
         private void RegisterButton_Click(object sender, RoutedEventArgs e)
         {
-            RegistrationWindow registrationWindow = new RegistrationWindow();
+            var registrationWindow = new RegistrationWindow();
+
             registrationWindow.Show();
         }
+
+        private void OnLoginButtonClicked(object sender, RoutedEventArgs e)
+        {
+            //var user = new User();
+            var twipsterMainWindow = new TwipsterMainWindow();
+
+            var validators = new List<IValidator>
+            {
+                new PasswordValidator()
+            };
+
+            try {
+                using (var context = new TwipsterDbContext())
+                {
+                    currentUser = context.Users.Single(x => x.Login == LoginTexBox.Text);
```

Comment:
You can use here coalescing operator ??= allowed in c# 8.0 (If you don't know how to use it don't use).